### ISSUE TITLE
Parachute installed, but cannot activate

### ISSUE THREAD

**Issue by psyphre** [tcorreabr/Parachute#83]
I've installed parachute and can see it activated in my kwin, but I cannot seem to trigger it.

ctrl+meta+d or ctrl+meta+z do nothing. I removed virtual desktop bar and restarted my pc so that it would not be a factor.

Using version 0.9.

I'm new to github and bug reporting so let me know if I am missing any important info.

Thanks!

**Comment by shifuharris** [comment]
I just installed Parachute both manually and via adding a new kwin script in system settings.  In both cases, I have the exact same issue as the OP. Tried various key combinations but not luck.  I also had the virtual desktop bar and also uninstalled it and rebooted.  

using version 0.9.0

Running on the following basic specs:

Lenovo Legion 5i
Pop!_OS 20.10
KDE Plasma: v5.19.5
KDE Frameworks: v5.74.0
QT Version: 5.124.2

Thanks
